docs/sdks/flutter/CHANGELOG.md

```diff
@@ -180,10 +180,10 @@ https://github.com/appwrite/appwrite/blob/master/CHANGES.md
 - Support for Appwrite 0.12
 - **BREAKING** Updated database service to adapt 0.12 API
 - **BREAKING** Custom ID support while creating resources
-- [View all the changes](https://github.com/appwrite/appwrite/blob/master/CHANGES.md#version-0120)
+- [View all the changes](https://github.com/appwrite/appwrite/blob/main/CHANGES.md#version-0120)
 
 ## 2.1.0
-- Updated `flutter_we_auth` plugin now supports Flutter web for OAuth2 sessions [read more](https://github.com/appwrite/sdk-for-flutter/blob/master/README.md#web)
+- Updated `flutter_we_auth` plugin now supports [Flutter web for OAuth2 sessions](https://github.com/appwrite/sdk-for-flutter/blob/master/README.md#web)
```

Comment:
#### Do not use `(click) here` links

For more information, see:
* https://www.w3.org/QA/Tips/noClickHere
* https://webaim.org/techniques/hypertext/link_text
* https://granicus.com/blog/why-click-here-links-are-bad/
* https://heyoka.medium.com/dont-use-click-here-f32f445d1021
```
(?i)(?:>|\[)(?:(?:click |)here|link|(?:read |)more)(?:</|\]\()
```